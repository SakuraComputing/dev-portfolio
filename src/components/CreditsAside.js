import React from 'react';

export const CreditsAside = () => (

    <aside className="credits aside section">
        <div className="section-inner">
            <h2 className="heading">Technologies Used</h2>
            <div className="content">
                <ul className="list-unstyled">
                    <li><a href="https://reactjs.org/" target="_blank"><i className="fa fa-external-link"></i>React JS</a></li>
                    <li><a href="https://angular.io/" target="_blank"><i className="fa fa-external-link"></i>Angular</a></li>
                    <li><a href="https://nodejs.org/en/" target="_blank"><i className="fa fa-external-link"></i>Node JS</a></li>
                    <li><a href="https://expressjs.com/" target="_blank"><i className="fa fa-external-link"></i>Express</a></li>
                    <li><a href="https://www.mongodb.com/" target="_blank"><i className="fa fa-external-link"></i>Mongo</a></li>
                    <li><a href="http://jquery.com/" target="_blank"><i className="fa fa-external-link"></i> jQuery</a></li>
                    <li><a href="http://getbootstrap.com/" target="_blank"><i className="fa fa-external-link"></i> Bootstrap</a></li>
                    <li><a href="http://sass-lang.com/" target="_blank"><i className="fa fa-external-link"></i> SASS</a></li>
                    <li><a href="http://fortawesome.github.io/Font-Awesome/" target="_blank"><i className="fa fa-external-link"></i> FontAwesome</a></li>
                </ul>
            </div>
        </div>
    </aside>
);

export default CreditsAside;


