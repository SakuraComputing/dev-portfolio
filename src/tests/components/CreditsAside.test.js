import React from 'react';
import { shallow } from 'enzyme';
import CreditsAside from '../../components/CreditsAside';

describe('CreditsAside Section', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<CreditsAside />);
        expect(wrapper).toMatchSnapshot();
    });
});