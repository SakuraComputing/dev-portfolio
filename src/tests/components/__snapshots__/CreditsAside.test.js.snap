// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditsAside Section should render correctly 1`] = `
<aside
  className="credits aside section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      Credits
    </h2>
    <div
      className="content"
    >
      <ul
        className="list-unstyled"
      >
        <li>
          <a
            href="http://getbootstrap.com/"
            target="_blank"
          >
            <i
              className="fa fa-external-link"
            />
             Bootstrap
          </a>
        </li>
        <li>
          <a
            href="http://fortawesome.github.io/Font-Awesome/"
            target="_blank"
          >
            <i
              className="fa fa-external-link"
            />
             FontAwesome
          </a>
        </li>
        <li>
          <a
            href="http://jquery.com/"
            target="_blank"
          >
            <i
              className="fa fa-external-link"
            />
             jQuery
          </a>
        </li>
        <li>
          <a
            href="https://github.com/IonicaBizau/github-calendar"
            target="_blank"
          >
            <i
              className="fa fa-external-link"
            />
             GitHub Calendar Plugin
          </a>
        </li>
        <li>
          <a
            href="http://caseyscarborough.com/projects/github-activity/"
            target="_blank"
          >
            <i
              className="fa fa-external-link"
            />
             GitHub Activity Stream
          </a>
        </li>
        <li>
          <a
            href="https://github.com/sdepold/jquery-rss"
            target="_blank"
          >
            <i
              className="fa fa-external-link"
            />
             jQuery RSS
          </a>
        </li>
      </ul>
      <hr />
    </div>
  </div>
</aside>
`;
